pkg/flowaggregator/s3uploader/s3uploader.go

```diff
@@ -260,70 +251,62 @@ func (p *S3UploadProcess) flowRecordPeriodicCommit() {
 			}
 			ctx, cancelFn := context.WithTimeout(ctx, bufferFlushTimeout)
 			defer cancelFn()
-			err := p.batchUploadAll(ctx, true)
+			err := p.batchUploadAll(ctx)
 			if err != nil {
 				klog.ErrorS(err, "Error when doing batchUploadAll on stop")
-			} else {
-				klog.V(4).InfoS("Total number of records uploaded to S3", "count", p.uploadedRec)
 			}
 			return
 		case <-p.uploadTicker.C:
-			err := p.batchUploadAll(ctx, true)
+			err := p.batchUploadAll(ctx)
 			if err != nil {
 				klog.ErrorS(err, "Error when doing batchUploadAll on triggered timer")
 			}
-		case <-logTicker.C:
```

Comment:
@antoninbas do you think we should add this periodic logging? Shall we log the number of records or the number of files? 

If records, one way I can think of is, make `bufferQueue =  [[buffer1, recordCount1], [buffer2, recordCount2], ...],`. As the upload may fail on any cached buffer, we cannot only have one var to keep track of the sum of the record counts for all buffers in the queue. Does this look good to you?

Reply:
I don't think this logging is very important, but I also don't understand your point about why a single variable is not sufficient. Why can't a single counter work? When upload succeeds, you increment the counter. When it fails, you don't.

Reply:
I was thinking given a buffer, is there an existing method to count how many rows are there? Otherwise, we may need to write a util function for this.

Reply:
↑ I was not saying about a single counter for "total success records". When you take out a buffer from the bufferQueue, and upload it successfully, how do we know how many records are in it.

Reply:
ok got it. I think you can wrap the `bytes.Buffer` with a struct:
```go
type fileBuffer struct {
    *bytes.Buffer
    int recordCount
}
```
and use that everywhere instead.